.github/workflows/main.yml

```diff
@@ -31,43 +37,66 @@ jobs:
 
     steps:
       - uses: actions/checkout@v1
-      - name: setup (linux)
-        if: startsWith(matrix.os, 'ubuntu')
-        run: |
-          sudo apt install cmake-data
-
-      # - name: setup (mac)
-      #   if: startsWith(matrix.os, 'macos')
-      #   run: |
-      #     brew update
-      #     brew upgrade
-      #     brew install cmake openssl
 
       - name: setup (windows)
         if: startsWith(matrix.os, 'windows')
         run: |
           choco install openssl cmake
+      
+      - name: setup (linux)
+        if: startsWith(matrix.os, 'ubuntu')
+        run: |
+          sudo apt install cmake-data
 
       - name: Create Build directory
-        run: cmake -E make_directory ${{runner.workspace}}/build
+        shell: bash
+        run: |
+          pwd
+          echo "$GITHUB_WORKSPACE/build"
+          cmake -E make_directory $GITHUB_WORKSPACE/build
+          cd $GITHUB_WORKSPACE/build
 
       - name: Configure CMake
         shell: bash
-        working-directory: ${{runner.workspace}}/build
         run: |
+          cd $GITHUB_WORKSPACE/build
           cmake $GITHUB_WORKSPACE \
           -DCOSE_C_USE_CONTEXT=${{matrix.USE_CONTEXT}} \
           -DCOSE_C_BUILD_DOCS=OFF \
           -DCOSE_C_COVERALLS=OFF \
           -DCOSE_C_BUILD_TESTS=ON \
           -DBUILD_SHARED_LIBS=OFF \
+          -DCMAKE_BUILD_TYPE=Debug \

```

Comment:
How is this going to interact with -DCOSE_C_OPTIMIZE=ON

Reply:
COSE_C_OPTIMIZE means optimize for size, I guess this option is intended for e.g. IoT devices.

what we want here, is to compile tests and run them... so, size optimization will probably not play a role...

this behavior is copied from appveyer... 
ctest needs on windows a build type to work as far as I see
do you see an issue with the current behaviour?

Reply:
No that is fine.  I just wanted to make sure that it should not cause any problems.  I added the one optimize build because I have some debug specific code for testing.